Solutions/SlashNext/Playbooks/SlashNextWebAccessLogAssessment/deploy.json

```diff
@@ -0,0 +1,717 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "PlaybookName": {
+      "defaultValue": "SlashNextWebAccessLogAssessment",
+      "type": "string"
+    }
+  },
+  "variables": {
+    "slashnext_connection_name": "[concat('slashnext-connection-', parameters('PlaybookName'))]",
+    "customApis_slashnext_name": "SlashNext"
+  },
+  "resources": [
+    {
+      "name": "SlashNext_azure_LinkedTemplate",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2018-05-01",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext/Playbooks/SlashNextWebAccessLogAssessment/azure_function_deploy.json"
```

Comment:
Do not include linked templates here. Include the actual resource in a single ARM template. This is applicable wherever the linked template is used.

Reply:
Kindly confirm this change is only required for the playbooks that are in child folders?

The main folder contains a playbook which deploys the whole package in the environment due to which we have used linked templates in it. So as per the sample solutions, my understanding is that only this arm template does not require the mentioned change set.

Reply:
@SaadatAbid-snx This change is required in the main deployment script also.

Reply:
@devikamehra updated!